### ISSUE TITLE
IPv6 addresses ?

### ISSUE THREAD

**Issue by olafleur** [gregjacobs/Autolinker.js#182]
Would it make sense to allow for IPv6 addresses to be clickable ?

For example, these addresses are not autolinked for the moment, but I think that they are valid IPv6 addresses :
http://[::1]/ (this is localhost)
http://[2001:4860:0:2001::68]/
http://[2001:4860:4860:0:0:0:0:8888]

**Comment by gregjacobs** [comment]
Yes, I think we should allow IPv6 addresses if they are preceded with `http://`. Would you want to work on this?

**Comment by olafleur** [comment]
Yes. :)
